site/src/xServices/workspace/workspaceXService.ts

```diff
@@ -410,9 +407,6 @@ export const workspaceMachine = createMachine(
         assign({
           cancellationMessage: undefined,
         }),
-      displayCancellationError: (context) => {
-        displayError(context.cancellationMessage)
```

Comment:
@presleyp Good point about the nature of the cancellation message. This was being displayed as an error before, so I just went with it. Do you think we should update this code to `displaySuccess` and remove it from the error list?

Reply:
Yeah if it's coming from an `onDone` then that makes sense